### ISSUE TITLE
Dropbox login with U2F

### ISSUE THREAD

**Issue by icetee** [buttercup/buttercup#233]
Hi, I tried authentication Dropbox with [U2F](https://www.yubico.com/product/yubikey-4-series/#yubikey-4), but can't recognize key. The console writes this:

`Unrecognized Content-Security-Policy directive 'worker-src'.`

`Refused to frame 'chrome-extension://kmendfapggjehodndflmmgagdbamhnfd/u2f-comms.html' because it violates the following Content Security Policy directive: "frame-src https://* carousel://* dbapi-6://* dbapi-7://* dbapi-8://* itms-apps://* itms-appss://* ".`

**Comment by sallar** [comment]
@icetee Sorry can you explain more about this U2F thing? What does it do and how is related to Dropbox?

**Comment by perry-mitchell** [comment]
@sallar @icetee I guess this has to do with browser integration with Yubikeys (and USB dongles in general)? I also have no idea how Yubikeys interact with the browser+dropbox, but I imagine it **may** be something we can address if Chrome supports it (depending on electron of course).

**Comment by icetee** [comment]
U2F use for two factor authentication. It's like mobile authentication, it's just a USB key.
Only supported by [Chrome / Opera](http://caniuse.com/#search=u2f).

It can be integrated into Github, Dropbox, etc.

I found this: https://github.com/electron/electron/issues/3226

Pop-up appears, click the USB button and copies a long key to it.

**Comment by perry-mitchell** [comment]
Thanks @icetee, this looks somewhat promising. That being said, we're currently blocked by this, so I'll place it on hold until upstream works out integration.
